data_handler.py

```diff
@@ -72,23 +79,27 @@ def regular_send(self, this_end, cpm, cpm_err):
         """
         Normal send
         """
-        try:
-            if self.manager.protocol == 'new':
-                self.manager.sender.send_cpm_new(this_end, cpm, cpm_err)
-                if self.queue:
-                    self.vprint(1, "Flushing memory queue to server")
-                while self.queue:
-                    trash = self.queue.popleft()
-                    self.manager.sender.send_cpm_new(
-                        trash[0], trash[1], trash[2])
-            else:
-                self.manager.sender.send_cpm(cpm, cpm_err)
-        except socket.error:
-            if self.manager.protocol == 'new':
-                self.send_to_queue(cpm, cpm_err)
-                self.vprint(1, "Socket error: saving to queue in memory")
-            else:
-                self.vprint(1, "Socket error: data not sent")
+        if self.led:
+            if self.led.blinker:
+                self.led.stop_blink()
+            self.led.on()
+        #try:

```

Comment:
You can remove these comments, but do include a comment saying that this can raise a socket.error which gets handled in DataHandler.